lib/install/config/webpacker.yml

```diff
@@ -34,6 +34,7 @@ development:
     host: localhost
     port: 8080
     https: false
+    hot_reloading: false
```

Comment:
@dhh, per the method change from `hot` to `hot_reloading`, I updated the yml file.